cesium_app/flask_server.py

```diff
@@ -246,35 +246,38 @@ def Features(featureset_id=None):
     try:
         if request.method == 'POST':
             data = request.get_json()
+            featureset_name = data.get('featuresetName', '')
+            datasetID = int(data['datasetID'])
+            features_to_use = [k[4:] for (k, v) in data.items()
```

Comment:
1. We're using `dataset_id` elsewhere, should we switch this everywhere or leave it the way it was? I think I prefer the old way.
2. You don't actually need to cast this, Peewee figures it out:
```
In [22]: m.Project.get(m.Project.id == '1')
Out[22]: <cesium_app.models.Project at 0x113aa3e80>
```